content/en/docs/concepts/storage/volumes.md

```diff
@@ -462,7 +462,8 @@ spec:
     required:
       nodeSelectorTerms:
       - matchExpressions:
-        - key: failure-domain.beta.kubernetes.io/zone
+        # failure-domain.beta.kubernetes.io/zone should be used prior to 1.21
+        - key: topology.kubernetes.io/zone
```

Comment:
This is not exactly deprecation, and should I think come in a separate commit / separate PR. Alternatively, revise the PR description and title.

Reply:
Done